@xen-orchestra/lite/src/components/Select.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles'
+import { FormControl, InputLabel, MenuItem, Select as SelectMaterialUi, SelectProps } from '@material-ui/core'
+import { withState } from 'reaclette'
+
+import IntlMessage from './IntlMessage'
+
+type AdditionalProps = Record<string, any>
+
+export type Options<T> = {
+  render: { (item: T, additionalProps?: AdditionalProps): React.ReactNode }
+  value: { (item: T, additionalProps?: AdditionalProps): string | ReadonlyArray<string> | number }
```

Comment:
Why `ReadonlyArray<string>`?

Reply:
Because that the expected type for the `value` prop of the `MenuItem` component.
`MenuItem` is using the `li` HTML tag.